### ISSUE TITLE
Bring in National Wetlands Inventory Data to RAVE or RS Context

### ISSUE THREAD

**Issue by joewheaton** [Riverscapes/RaveAddIn#132]
As per email with @wally-mac, @tyler1218hatch please work with @philipbaileynar as necessary to bring in the [National Wetlands Inventory Data](https://www.fws.gov/wetlands/data/web-map-services.html) into either Riverscapes Context project, or if the web mapping service is reliable, as a default layer in the Basemaps (preferred):
![image](https://user-images.githubusercontent.com/9723905/113898555-375baa00-9789-11eb-9d4b-281bfa93f8c8.png)

According to their documentation:

> Launch ArcCatalog.
> Under the “GIS Servers” directory in the explorer window on the left, double click on “Add WMS Server” to open the Add WMS Server dialog box.
> Enter the following WMS address in the URL text box: 
> 
> Wetlands Data WMS address:
> https://www.fws.gov/wetlands/arcgis/services/Wetlands/MapServer/WMSServer? 
> Wetlands Raster WMS address::
> https://www.fws.gov/wetlands/arcgis/services/Wetlands_Raster/ImageServer/WMSServer? 
> Wetlands Status WMS address:
> https://www.fws.gov/wetlands/arcgis/services/Wetlands_Status/MapServer/WMSServer? 
> Riparian Data WMS address:
> https://www.fws.gov/wetlands/arcgis/services/Riparian/MapServer/WMSServer?   
> 
> Click on the “Get Layers” button to view the service and layer information.
> Click on the “OK” button to add the connection and view it in ArcCatalog. You will see the connection show up under “GIS Servers.”
> This WMS connection will now also be available in ArcMap under “GIS Servers” after you select “add data”.

**Comment by joewheaton** [comment]
![image](https://user-images.githubusercontent.com/9723905/113899508-1c3d6a00-978a-11eb-84ca-985f4cf50455.png)

From: https://www.fws.gov/wetlands/arcgis/rest/services/WetlandsTopo/WetlandsTopoService/MapServer

**Comment by tyler1218hatch** [comment]
@joewheaton I'll get started on this soon!

**Comment by tyler1218hatch** [comment]
@philipbaileynar 
I've got these loading properly into ArcMap. I'm assuming the next step is to change the business logic XML so it can access these datasets. What's the syntax for accessing a web server within the Business Logic? Thanks in advance for the help, I'm pretty inexperienced with this kind of web mapping.

**Comment by tyler1218hatch** [comment]
@philipbaileynar Okay, I found the Basemaps XML that shows examples of these. I'll get started on the pull request, sorry for the extra notifications.

**Comment by joewheaton** [comment]
So now that we have QRAVE (and I gather this #137 list for updating ArcRAVE similar to Q), I am confused as to where an edit like this one should go @MattReimer and @philipbaileynar? I'm thinking it may need to be split out into some stuff in RiverscapesXML and some here and some in [QRAVEPlugin](https://github.com/Riverscapes/QRAVEPlugin)? 

- [ ] Ideally, changes to Basemaps would be made once in [RiverscapesXML/../`BaseMaps.xml`](https://github.com/Riverscapes/RiverscapesXML/blob/master/BaseMaps.xml) and then eventually that fixes basemap menus in BOTH QRAVE and ArcRAVE. @MattReimer is that right?
- [ ] If correct, should the #134 be closed or cancelled or retracted or something, and the edits (taking on board my review comments) moved to an edit to [RiverscapesXML/../`BaseMaps.xml`](https://github.com/Riverscapes/RiverscapesXML/blob/master/BaseMaps.xml) and a new pull request by Tyler in that repo? If so, that would mean edits don't get reflected into Arc until it is also pointing at RiverscapesXML.
- [ ] Except in very rare circumstances, @MattReimer I am assuming that we do not want nor expect to offer the same Basemap functionality and richness in WebRAVE as we do in other flavors?

**Comment by MattReimer** [comment]
@joewheaton you're right. Until ArcRave catches up to use the changes we made in RiverscapesXML we will need to make the changes in both places.

I'm not sure what kind of timeline you'd like to see but given that we have to do another release of ArcRave just to get these changes it might make sense to wait. Make the changes to RiverscapesXML and they will come over once the work on ArcRave is done. I'll defer to @philipbaileynar on that though.

We can do what we want in WebRave but every complexity has to be coded from scratch so I want us to be very careful and deliberate about what goes over. ESRI webmaps get really noisy with all the layers and it makes it harder to use.  I think implementing all the baselayers from BaseMaps.xml in webrave is overkill. Maybe we could scale it back to just the essentials. Happy to discuss though.

I'd have to look at the licenses for all the layer files. We may have to attribute them at the bottom of the screen or something.

**Comment by joewheaton** [comment]
Let's wait until we have timeline and strategy for Arc updates. No need to waste time with unnecessary temporary fixes.
